### DISCUSSION TITLE
2022/11/02/010_MysqlInstall/

### DISCUSSION THREAD

**Discussion by giscus** [XLJZT/XLJZT.github.io#15] [Announcements]
# 2022/11/02/010_MysqlInstall/

Ubuntu系统下MySQL的安装及所遇问题的解决办法 安装直接命令行安装 1sudo apt-get install mysql-server  根据提示完成安装。安装查询是否成功 1mysql -V   大写的V  查看mysql服务状态 1systemctl status mysql.service  登录我安装的版本为 8.0.31  问题： 由于在安装过程中没有提示输入root的密码，导

http://localhost:4000/2022/11/02/010_MysqlInstall/

**Comment by XLJZTS** [comment]
test
